### PR TITLE
Update blueberries.css

### PR DESCRIPTION
**Pull request by HugoBDesigner**
Fixed .language-dropdown__dropdown__results__item rule, improved readability in some rules, added new rules.

Please review the "improved readability" rules specifically, as they might look different in your instance.

### CODE REVIEW

File: blueberries.css [hunk 1]
```diff
@@ -539,6 +548,13 @@
         border: 1px solid hsl(var(--hue), 16%, 45%);
     }
 
+    /*  Apparently an HTML "select" element cannot be properly styled in its expanded form.
+        So this exists to mitigate contrast issues with the (unstylable) selected item. */
+    .compose-form__poll-wrapper select option {
+        background-color: hsl(var(--hue), 16%, 19%);
+    /*     color: hsl(var(--hue), 17%, 15%); */
+
```
Comment (brodokk) [inline blueberries.css:557]: I think their is a `}` missing around here.

Be careful maybe more code can be missing too!

**Review by brodokk — CHANGES_REQUESTED** [review]

**Comment by brodokk** [comment]
The language dropdown seems nearly good:

Before:

image

After (the `e` is still in the search bar but not visible):

image

**Comment by brodokk** [comment]
The search bar for the emoji have also the same problem than the the language dropdown

image

**Comment by brodokk** [comment]
Everything else seems good! 👍

File: blueberries.css (hunk 1 above)
Comment (HugoBDesigner) [reply blueberries.css:557]: Fixed it, thanks for letting me know! I have also looked more carefully into all other rules for potential similar issues and haven't found any other.

**Comment by HugoBDesigner** [comment]
Thanks for reviewing this PR! I fixed the missing bracket, reviewed the overall CSS for potential issues, and improved readability on search bars.

**Comment by brodokk** [comment]
Sounds good to me! Thanks again for this PR!

**Pull request merged**